.github/workflows/deploy-ecamp3-logging.yml

```diff
@@ -25,39 +33,43 @@ jobs:
 
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
 
-      - name: Dump secrets to .env
+      - name: Dump secrets to /tmp/secrets.yaml
         run: |
-          echo '${{ toJSON(secrets) }}' | jq -r 'keys[] as $k | select(.[$k] |contains("\n") | not) | "\($k)=\"\(.[$k])\""' >> .env
-        working-directory: .ops/ecamp3-logging
+          cat << 'EOF' | tee -a /tmp/secrets.yaml
+          ${{ toJSON(secrets) }}
+          EOF
+          jq '.' /tmp/secrets.yaml
```

Comment:
Why the jq '.'? Is that just pretty printing? Is that necessary or just for debugging?

Reply:
this is pretty printing for debugging.
It is json, but json is a subset of yaml.